### ISSUE TITLE
Fix S1313 FN: Support utf-8 literals

### ISSUE THREAD

**Issue by gregory-paidis-sonarsource** [SonarSource/sonar-dotnet#6376]
### Description

FN for S1313 (HardcodedIpAddress).

Using C#11 utf-8 string literals, the rule is not raising an issue.

### Repro steps

```csharp
            var ip2 = "192.168.0.1"u8; // FN
            var ip3 = """192.168.0.1"""u8; // FN
            var ip4 = """
                192.168.0.1
                """u8; // FN

```

### Expected behavior

The rule should raise an issue.

### Actual behavior

The rule does not raise an issue.
